Group Finder Documentation Update (#105)
KingdomFirst/RockBlocks: Groups/GroupFinder_Readme.md

> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;8&nbsp;&nbsp;</span>**Show All Groups** When set to yes, all groups will show including those where Is Public is set to false.  This is most often used on a staff internal page.
>
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;9&nbsp;&nbsp;</span>**Single Select Campus Filter** When set to yes, the campus filter will be a drop down instead of checkbox.

## Advanced

**How to Set Attribute Filters via URL Parameters**

> You can link to fill out attribute filters, or in combination with the *Auto Load* setting run a search automatically, via a URL Parameter. To use this capability the parameters are in the format `filter_<attributeKey>_<fieldTypeId>=<value(s)>` (i.e. /page/236?filter_GroupCategory_16=609)

Reviewer: ```suggestion
Attribute filters can be automatically filled out using URL Parameters. If used in combination with the *Auto Load* setting, a search can be run automatically. To use this capability the parameters are in the format `filter_<attributeKey>_<fieldTypeId>=<value(s)>` (i.e. /page/236?filter_GroupCategory_16=609)
```
